.env.lacchain

```diff
@@ -61,7 +72,7 @@ REACT_APP_EOS_API_NETWORK_NAME=lacchain
 REACT_APP_EOS_API_NETWORK_LABEL=LACChain
 REACT_APP_EOS_API_NETWORK_LOGO=https://ipfs.eosio.cr/ipfs/QmdfvY1gJk688Sxd7iTg6Dbd5VQTv9Mr1M21Fo7nvJapRv
```

Comment:
no longer using IPFS for this , use this URL instead

https://eosio.online/images/lacchain.jpg